examples/2_intermediate/mta.ipynb

```diff
@@ -0,0 +1,495 @@
+       ":param has_delay_alert: A boolean that is True iff a train currently has a delay alert published\n",
+       ":return: A list of `Trip` objects\n",
+       "\u001b[0;31mFile:\u001b[0m      ~/micromamba/envs/csp-vanilla/lib/python3.11/site-packages/nyct_gtfs/feed.py\n",
+       "\u001b[0;31mType:\u001b[0m      method"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "feed.filter_trips?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "9d89289d-ee10-4bee-a37d-0863fde7d1f9",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[0;31mType:\u001b[0m        property\n",
+       "\u001b[0;31mString form:\u001b[0m <property object at 0x7f37ac4d25c0>\n",
+       "\u001b[0;31mDocstring:\u001b[0m   Get the list of subway trips from the GTFS-realtime feed. Returns a list of `Trip` objects"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "feed.trips?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bf0fe58-ced8-49a7-84bb-09b9abdb41ee",
+   "metadata": {},
+   "source": [
+    "In our case, we will (for simplicity) filter the trips to collect information only about the 1, 2 and 3 trains, and we will start with trains going through 34 St-Penn Station."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "429163be-6a02-4a14-b1f5-5f5016213af0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{\"069950_1..N03R\", STOPPED_AT 133N @12:52:08},\n",
+       " {\"070550_1..N03R\", STOPPED_AT 137N @12:51:41},\n",
+       " {\"071150_1..N03R\", IN_TRANSIT_TO 139N @12:52:23},\n",
+       " {\"067150_2..N09R\", STOPPED_AT 128N @12:51:58},\n",
+       " {\"067950_2..N01R\", IN_TRANSIT_TO 132N @12:52:26},\n",
+       " {\"068750_2..N09R\", IN_TRANSIT_TO 230N @12:52:27},\n",
+       " {\"069550_2..N01R\", STOPPED_AT 235N @12:52:05},\n",
+       " {\"070350_2..N09R\", STOPPED_AT 241N @12:51:56},\n",
+       " {\"071150_2..N01R\", IN_TRANSIT_TO 246N @12:52:22},\n",
+       " {\"066500_3..N01R\", STOPPED_AT 132N @12:51:45},\n",
+       " {\"067100_3..N01R\", STOPPED_AT 137N @12:51:58},\n",
+       " {\"067900_3..N01R\", INCOMING_AT 230N @12:52:26},\n",
+       " {\"068700_3..N01R\", IN_TRANSIT_TO 233N @12:52:22},\n",
+       " {\"069500_3..N01R\", INCOMING_AT 239N @12:52:25},\n",
+       " {\"070300_3..N01R\", STOPPED_AT 252N @12:51:38},\n",
+       " {\"071150_3..N01R\", INCOMING_AT 256N @12:52:06}]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# This cell can be run multiple times, and data will be refreshed every 30s\n",
+    "feed.refresh()\n",
+    "trains = feed.filter_trips(underway=True, headed_for_stop_id=['128', '128N'])\n",
```

Comment:
You also need `128S` here, right? `128N` will pick up Uptown trains and `128S` will pick up Downtown. Let's also make a note in the text of where we got this `stop_id` from (the stops.txt file provided by the MTA).

Reply:
Thanks - I got that from inspecting the data itself since I don't have access to the documentation for the API, but I'll at least make a note.